barretenberg/cpp/pil/vm2/alu.pil

```diff
@@ -8,8 +8,9 @@ namespace alu;
 pol commit sel;
 
 pol commit sel_op_add;
-pol commit sel_op_lt;
 pol commit sel_op_eq;
+pol commit sel_op_lt;
+pol commit sel_op_lte;
```

Comment:
Sorry about the diffs - I just rearranged the ops to match the wire ordering (IIRC add, eq, lt, lte) for consistency here and in a few places below